src/main/java/org/hawkular/client/android/backend/BackendClient.java

```diff
@@ -211,16 +212,13 @@ public void getAlerts(@NonNull Date startTime, @NonNull Date finishTime, @NonNul
 
     public void acknowledgeAlert(@NonNull Alert alert,
                                  @NonNull Callback<List<String>> callback) {
-        URI uri = Uris.getUri(String.format(BackendPipes.Paths.ALERT_ACKNOWLEDGE, alert.getId()));
-
-        readPipe(BackendPipes.Names.ALERT_ACKNOWLEDGE, uri, callback);
+        alert.setId(alert.getId());
```

Comment:
this one line I was forced to do. IDE is somehow eating the setter method if it is not used. And causing program to fail. Not leting me live even after setting minifyEnabled false